dot/sync/test_helpers.go

```diff
@@ -88,3 +91,43 @@ func BuildBlock(t *testing.T, instance runtime.Instance, parent *types.Header, e
 		Body:   body,
 	}
 }
+
+const (
+	defaultMinPeers     = 1
+	defaultMaxPeers     = 5
```

Comment:
nit Define these two as inlined constants where they're used in `newTestChainSyncWithReadyBlocks`

Reply:
Updated.